jcl/src/jdk.management/share/classes/com/ibm/lang/management/ThreadMXBean.java

```diff
@@ -75,13 +87,20 @@ public long getNativeThreadId(long threadId)
 	 *					currently locked ownable synchronizers is to be included in
 	 *					the returned array
 	 * @return  Array of ExtendedThreadInfo objects.
+	/*[IF JAVA_SPEC_VERSION < 24]
 	 * @throws SecurityException is thrown if the caller does not have sufficient permissions
 	 * (ManagementPermission("monitor"))
+	/*[ENDIF] JAVA_SPEC_VERSION >= 24
 	 * @throws UnsupportedOperationException is thrown if the JVM does not support monitoring
 	 * object monitor usage or ownable synchronizer usage, even as it has been specified.
 	 * @throws InternalError is thrown in case an error occurs while fetching thread information,
 	 * typically, an internal error resulting from an inconsistency in the class library.
 	 */
 	public ExtendedThreadInfo[] dumpAllExtendedThreads(boolean lockedMonitors, boolean lockedSynchronizers)
-			throws SecurityException, UnsupportedOperationException, InternalError;
+			throws
+			InternalError,
+			/*[IF JAVA_SPEC_VERSION < 24]*/
+			SecurityException,
+			/*[ENDIF] JAVA_SPEC_VERSION < 24 */
+			UnsupportedOperationException;
```

Comment:
Suggest the `throws` keyword should be indented less than the exception names. Either indent line 100 one tab less, or lines 101-105 one tab more.